docker-entrypoint.sh

```diff
@@ -4,12 +4,9 @@ set -e
 
 umask $UMASK
 
-usermod -u $PUID user
-groupmod -g $PGID users
-
-chown -R user:users /app
-chown -R user:users /rec
+chown -R ${PUID}:${PGID} /app
+chown -R ${PUID}:${PGID} /rec
 
 export HOME=/home/user
 
-exec /usr/local/bin/gosu user dotnet /app/BililiveRecorder.Cli.dll $@
+exec /usr/local/bin/gosu ${PUID}:${PGID} dotnet /app/BililiveRecorder.Cli.dll $@
```

Comment:
```bash
#!/bin/bash

set -e

umask "$UMASK"

if [ "$(id -u)" != "0" ]; then
    echo "Skipped changing user and group because current user is not root."
    exec dotnet /app/BililiveRecorder.Cli.dll "$@"
fi

PUID=${PUID:-0}
PGID=${PGID:-0}

if [ "${PUID}" != "0" ] && [ "${PGID}" != "0" ]; then
    chown -R "${PUID}":"${PGID}" /app
    chown -R "${PUID}":"${PGID}" /rec
    exec /usr/local/bin/gosu "${PUID}":"${PGID}" dotnet /app/BililiveRecorder.Cli.dll "$@"
else
    echo "PUID or PGID is empty or zero, running as root."
    exec dotnet /app/BililiveRecorder.Cli.dll "$@"
fi
```

这样如何？改动太大没法直接 suggest changes.

Reply:
感觉好像没必要呢，在Dockerfile里面已经默认UID和GID为0了，也就是说默认不设置就是使用root权限运行程序，似乎没必要再判断一次了

Reply:
这里再设置一次是为了下面的判断，不然写成 `if [ -n "${PUID}" ] && [ "${PUID}" != "0" ] && [ -n "${PGID}" ] && [ "${PGID}" != "0" ]; then`